R/az_hourly.R

```diff
@@ -91,13 +91,33 @@ az_hourly <- function(station_id = NULL, start_date_time = NULL, end_date_time =
       as.numeric
     )) %>%
     dplyr::filter(.data$meta_station_id != "az99") %>%
-    dplyr::mutate(date_datetime = lubridate::ymd_hms(.data$date_datetime)) %>%
+    dplyr::mutate(
+      date_datetime =
+        lubridate::force_tz(
```

Comment:
but here, `ymd_hms()` _doesn't_ see a timezone and you need `force_tz()` to keep the time and only change the timezone.  Nice!